express/services/playerTeamSeasons.js

```diff
@@ -0,0 +1,16 @@
+const models = require('../../sequelize/models');
+
+function getAll(playerId) {
+    return models.playerTeamSeason.findAll({
+        where: {
+            playerId
+        },
+        include: [{
+            model: models.result
```

Comment:
I think it will be a more flexible solution if we allow specifying the included relations by sending them to service. What do you think about this?